docs/reference/2-ods-api/platform-dev-guide/overview.md

```diff
@@ -124,17 +124,58 @@ PostgreSQL ODS.
 
 A high-level view looks something like this:
 
-![High Level](../../../../static/img/reference/ods-api/image2020-10-20_11-33-39.png)
+```mermaid
+block-beta
```

Comment:
Although this is a "beta" feature in Mermaid, it does render correctly in Docusaurus. Not all beta diagrams work - for example, the architecture diagram does not work here.

Reply:
Adding as a note for our migration.